medusa/__main__.py

```diff
@@ -777,6 +777,9 @@ def initialize(self, console_logging=True):
             app.TRAKT_SYNC = bool(check_setting_int(app.CFG, 'Trakt', 'trakt_sync', 0))
             app.TRAKT_SYNC_REMOVE = bool(check_setting_int(app.CFG, 'Trakt', 'trakt_sync_remove', 0))
             app.TRAKT_DEFAULT_INDEXER = check_setting_int(app.CFG, 'Trakt', 'trakt_default_indexer', 1)
+            if app.TRAKT_DEFAULT_INDEXER == 3:
```

Comment:
This is the kind of code that gets forgotten at some time and we wonder why we always have indexer 1 selected instead of 3. Why not add a warning saying that it should be changed, just for 1 release and remove it after that?